README.md

```diff
@@ -1 +1,134 @@
-# Budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget-app <a name="about-project"></a>
+
+**Budget App** keeps track of all your purchases organized in categories of your creation. It allows you to create new categories and create new expenses for each category. In order to use the app, a user must register. 
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- CSS
+- HTML
+- JavaScript
+- Ruby 
+- Ruby on Rails
+- Postgresql
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create categories**
+- **Create expenses**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can view a live demo of this application via this [link](https://saveit-usxc.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need Ruby and Postgresql installed on your computer.
+
+### Setup
+
+Clone this repository to your desired folder: 
+
+`git clone https://github.com/laiifuu/Budget-app.git`
+
+
+### Install
+
+Then install the project's necessary dependencies with: 
+
+`bundle install`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hayet Fellah**
+
+- GitHub: [@laiifuu](https://github.com/laiifuu)
+- Twitter: [@hayet_fe](https://twitter.com/hayet_fe)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add the options to delete a category and expense**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give this project a ⭐️ if you like it !
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)for this project,  "`The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.`", please kindly acknowledge [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for his original design idea.